pkg/sql/schemachanger/scbuild/internal/scbuildstmt/drop_index.go

```diff
@@ -59,38 +46,43 @@ func DropIndex(b BuildCtx, n *tree.DropIndex) {
 			pgnotice.Newf("CONCURRENTLY is not required as all indexes are dropped concurrently"))
 	}
 
+	anyIndexesDropped := false
 	for _, index := range n.IndexList {
-		dropAnIndex(b, index, n.IfExists, n.DropBehavior)
-
+		dropped := dropAnIndex(b, index, n.IfExists, n.DropBehavior)
+		if dropped {
+			anyIndexesDropped = true
+		}
```

Comment:
nit: replace this with `anyIndexesDropped = dropAnIndex(b, index, n.IfExists, n.DropBehavior)`

Reply:
It's possible that if multiple indexes are specified one of them won't be dropped. Is the following cleaner:
```
		anyIndexesDropped = anyIndexesDropped ||
			dropAnIndex(b, index, n.IfExists, n.DropBehavior)
```

Reply:
it can be `anyIndexesDropped = dropAnIndex(b, index, n.IfExists, n.DropBehavior) || anyIndexesDropped` but that's probably too clever. I'm cool with this. If you did feel the need to do anything more clever:

```
didDrop := dropAnIndex(/* ... *./)
anyIndexesDropped = anyIndexesDropped || didDrop
```